### PR TITLE
Prevents an error when value is an integer

### PR DESCRIPTION
**Pull request by javierarce**
Converts the model value to string in case there was an integer stored.

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4334/))

**Comment by saleiva** [comment]
bump!

**Comment by saleiva** [comment]
re-bump!

**Comment by javierarce** [comment]
@xavijam || @javisantana: can you review this, please?

**Comment by saleiva** [comment]
re-re-bump!

**Comment by javierarce** [comment]
@xavijam: can you have a look? I asked for a test case but didn't get info :|

File: lib/assets/javascripts/cartodb/table/menu_modules/legend_editor.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ cdb.admin.mod.LegendEditorItem = cdb.core.View.extend({
       var tempModel = new cdb.core.Model();
 
       var value  = this.model.get("value") || "";
-      var hasURL = value.indexOf("http") != -1;
+      var hasURL = value.toString().indexOf("http") != -1;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/menu_modules/legend_editor.js]: Is it worth it to use ```!== ```?

Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/menu_modules/legend_editor.js]: Sure, I'll change it.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4688/))

**Comment by saleiva** [comment]
re-re-re-bump!

**Comment by xavijam** [comment]
Go go go go!

**Comment by xavijam** [comment]
:es:

**Pull request merged**